homework.md

```diff
@@ -92,19 +85,19 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save code to Github
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+We use Visual Studio code to write code and Javascript as our language. We also use Css and Html.
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To record the changes made to the code and to let others know what was changed and when.
 
 ## Question 18
```

Comment:
any other places you can rely on if you need help with Git?

Reply:
Documentation https://docs.github.com/en
google

Reply:
Same with this